pkg/engine/fetchit.go

```diff
@@ -331,7 +324,7 @@ func (f *Fetchit) RunTargets() {
 		ctx, cancel := context.WithCancel(context.Background())
 		defer cancel()
 		mt := method.GetKind()
-		klog.Infof("Processing Target: %s Method: %s Name: %s", method.GetTarget().name, mt, method.GetName())
+		klog.Infof("Processing git target: %s Method: %s Name: %s", method.GetTarget(), mt, method.GetName())
```

Comment:
With this change the log message is getting slightly messed up

```
I0725 17:07:26.906196       1 fetchit.go:327] Processing git target: &{https://github.com/containers/fetchit   main {%!s(int32=0) %!s(uint32=0)} %!s(bool=false)} Method: raw Name: raw-ex
```

Reply:
updated, good catch thanks